README.md

```diff
@@ -1,176 +1,34 @@
-# Opni = AIOps for Kubernetes + Observability Tools
-
+# Multi Cluster Observability with AIOps 
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/rancher/opni)
 
-Opni currently features log anomaly detection for Kubernetes.
-
-##### What does Opni give me?
-* AI generated insights on your cluster's log messages
-  * **Control Plane & etcd** insights
-    * Pretrained models maintained by Rancher Labs
-* Every log message sent to Opni will be marked as:
-  * **Normal**
-  * **Suspicious** - Operators may want to investigate
-  * **Anomalous** - Operators definitely should investigate  
-* Opensearch + Opensearch Dashboards
-  * Opni dashboard to consume log insights & explore logs 
-
-![alt text](https://opni-public.s3.us-east-2.amazonaws.com/opni-inside-cluster-diagram.png)
-
-----
-
-### Deprecation Notice
-  - GPU Learning is temporarily disabled in the v0.4.0 release as Opni moves to a multicluster architecture.  This will be returning in a future release
-  - The v1beta1 API has been deprecated in this release.  Please migrate to v1beta2.
-  - The UI and Insights services, which were experimental, have been removed
+The three pillars of observability are ***logs, metrics and traces***.
+The collection and storage of observability data is handled by observability backends and agents.
```

Comment:
this phrasing seems slightly off to me, in context. Maybe move this sentence down further.